build.gradle

```diff
@@ -37,5 +37,10 @@ run {
 
     /* Pass all the properties to the application java process*/
     systemProperties System.getProperties()
+
+    if (System.getProperty('debug') != null) {
```

Comment:
This is fine for now, but we'll want to make this a project property rather than a system property eventually.